tes-adapter/src/main/java/com/epam/pipeline/tesadapter/configuration/CustomConverter.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.pipeline.tesadapter.configuration;
+
+import org.springframework.boot.autoconfigure.http.HttpMessageConverters;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+import java.util.Collections;
+
+@Configuration
+public class CustomConverter implements WebMvcConfigurer{
```

Comment:
Please add, a space after all curly braces.
I'll check our checkstyle configuration, it shall be reported as all other warnings.